docs/pages/infrastructure/dns-and-domain-registration-security.mdx

```diff
@@ -0,0 +1,395 @@
+- Spammers harvesting contact details
+- Competitors researching your infrastructure
+- Anyone running a simple WHOIS lookup
+
+**Setup**:
+- Enable WHOIS privacy/proxy service through your registrar (often free or low-cost)
+- Use company information instead of personal details where privacy isn't available
+- Consider using a separate business entity for domain registration
+- Be aware that some TLDs (.us, .ca) don't allow WHOIS privacy
+
+**Important**: WHOIS privacy doesn't affect your legal ownership - you remain the legitimate owner, the privacy service just shields your personal information from public view.
+
+#### Domain Expiration Protection
+Domain expiration is a critical yet often overlooked security risk. When domains expire, they enter a grace period before becoming publicly available, creating an opportunity for attackers to snipe your domain.
+
+**Expiration timeline**:
+- Day 0: Domain expires (site goes down)
+- Day 1-45: Auto-renew grace period (can renew at normal price)
+- Day 46-75: Redemption period (costs 10x+ to recover)
+- Day 76-80: Pending delete
+- Day 81: Public availability (bot armies compete to register)
+
+**Protection measures**:
+- **Enable auto-renewal** on all critical domains
+- **Set multiple renewal reminders** at 90, 60, 30, and 7 days before expiration
+- **Register domains for maximum period** (up to 10 years for most TLDs)
+- **Use a domain monitoring service** that alerts on upcoming expirations
+- **Document renewal dates** in your security calendar
+- **Ensure payment methods stay current** - expired credit cards are a common cause of accidental expiration
+- **Designate a backup person** responsible for domain renewals
+
+**Pro tip**: Set calendar reminders to check auto-renewal status quarterly - don't assume it's working until you verify.
+
+## Monitoring and Detection
+
+### 1. DNS Record Monitoring
+DNS record monitoring involves continuously checking your domain's DNS records for unauthorized changes. Attackers often modify DNS records to redirect traffic to malicious servers while keeping your site partially functional.
+
+**What to watch for**:
+- **Nameserver (NS) record changes**: Attackers change your nameservers to point to their own DNS servers, giving them complete control over all DNS records
+- **TTL values dropping below 60 seconds**: Very low TTLs can indicate preparation for rapid DNS changes during an attack
+- **CAA record removal**: Allows any Certificate Authority to issue certificates for your domain
+- **DNSSEC disabled unexpectedly**: Removes cryptographic protection from DNS responses
+
+**If nameservers remain unchanged, also monitor**:
+- **A/AAAA record modifications**: IP address changes could redirect users to malicious sites
+- **MX record modifications**: Email server changes could intercept password reset emails
+- **TXT record changes**: Could affect email security (SPF/DMARC) or domain validation
+
+**Tools for DNS monitoring**:
```

Comment:
This lists monitoring tools, but you list dnsviz and dnsdumpster, which are not for monitoring. 

I just looked for some other interesting tools, and I've found zone control tools like OctoDNS and DNSControl that support GitOps workflows and auditable changes